web/pages/Chat/components/InputBar.tsx

```diff
@@ -161,74 +165,72 @@ const InputBar: Component<{
         onSubmit={() => send()}
         cleared={cleared}
       />
-      <div>
-        <button
-          onClick={onButtonClick}
-          class="rounded-l-none rounded-r-md border-l border-[var(--bg-700)] bg-[var(--bg-800)] py-2 px-2 hover:bg-[var(--bg-700)]"
-        >
-          <MoreHorizontal />
-        </button>
-        <DropMenu show={menu()} close={() => setMenu(false)} vert="up" horz="left">
-          <div class="flex w-48 flex-col gap-2 p-2">
-            {/* <Button schema="secondary" class="w-full" onClick={generateSelf} alignLeft>
+      <button
+        onClick={onButtonClick}
+        class="h-full rounded-l-none rounded-r-md border-l border-[var(--bg-700)] bg-[var(--bg-800)] py-2 px-2 hover:bg-[var(--bg-700)]"
```

Comment:
Addition of `h-full` is the only change, everything else is whitespace from removing a layer of nesting.